turbopack/crates/turbopack-ecmascript/tests/analyzer/graph/turbopack-ignore/input.js

```diff
@@ -0,0 +1,4 @@
+const w_i = import(/* webpackIgnore: true */ "w_i");
+const t_i = import(/* turbopackIgnore: true */ "t_i");
+const w_r = require(/* webpackIgnore: true */ "w_r");
+const t_r = require(/* turbopackIgnore: true */ "t_r");
```

Comment:
A test case where there's both a `: false` and a `: true` directive might be good to verify the logic that checks for only the last valid directive.

Reply:
Also a test case where there's an import or require inside of an uncommented function or a control block.

Reply:
Will add the first comment, as for your second comment there are more tests in the next PR in the stack.